source/_patterns/01-atoms/grid/demo/layout-demo.twig

```diff
@@ -0,0 +1,17 @@
+<h1>Layout Demo</h1>
+<br>
+<div class="container bg-secondary" style="height: 20rem;">
+  <h1 class="text-white">Container Standard</h1>
+  <p>
+    Containers are the most basic layout element in Bootstrap and are required when using our default grid system.
+    Choose from a responsive, fixed-width container (meaning its <code>max-width</code> changes at each breakpoint) or fluid-width (meaning it’s <code>100%</code> wide all the time).
+    While containers can be nested, most layouts do not require a nested container.
```

Comment:
I don't think we can have containers within containers, or at least it is highly recommended against.